README.md

```diff
@@ -148,6 +160,8 @@ moment("2010-10-20 4:30", "YYYY-MM-DD HH:mm");
 import parse from "date-fns/parse";
 parse("2010-10-20 4:30", "yyyy-MM-dd H:mm", new Date());
 // => "2010-10-19T17:30:00.000Z"
+
+// dayjs ❌ not support custom format parse
```

Comment:
I don't think you need to mention `dayjs` if it's not supported

Reply:
I think this comparison would help people make a decision when choosing date lib to replace moment.js.

Reply:
`not support` => `do not support`

Reply:
@ddvkid then we should list the pros and cons of each lib.